druid/src/widget/container.rs

```diff
@@ -170,6 +207,11 @@ impl<T: Data> Widget<T> for Container<T> {
             ctx.request_paint();
         }
         self.child.update(ctx, data, env);
+            if let Some(brush) = self.foreground.as_mut() {
+            trace_span!("update foreground").in_scope(|| {
+                brush.update(ctx, old_data, data, env);
+            });
+        }
```

Comment:
I don't think this needs to be after `self.child.update`. Let's move it immediately after the background update block.